src/nbl/asset/utils/CJITIncludeLoader.cpp

```diff
@@ -0,0 +1,61 @@
+
+#include "CJITIncludeLoader.h"
+
+
+CJITIncludeLoader::CJITIncludeLoader()
+{
+    m_includes["device_capabilities.hlsl"] = collectDeviceCaps();
```

Comment:
it would seem you obviously need the `const SPhysicalDeviceLimits&` and `const SPhysicalDeviceFeatures&` to collect the caps

Reply:
Also make the key `"nbl/builtin/hlsl/jit/device_capabilities.hlsl"` such that you can match it with `searchPath / includeName`